content/en/user-manual/api/asset-list.md

```diff
@@ -17,21 +17,23 @@ Get the details of all assets in a project for a specific branch
 ## Example
 
 ```none
-curl -H "Authorization: Bearer {accessToken}" "https://playcanvas.com/api/projects/{projectId}/assets?branchId={branchId}"
+curl -H "Authorization: Bearer {accessToken}" "https://playcanvas.com/api/projects/{projectId}/assets?branchId={branchId}&skip={number}&limit={number}"
 ```
 
 HTTP Request
 
 ```text
-GET https://playcanvas.com/api/projects/{projectId}/assets?branchId={branchId}
+GET https://playcanvas.com/api/projects/{projectId}/assets?branchId={branchId}&skip={number}&limit={number}
 Authorization: Bearer {accessToken}
 ```
 
 ## Parameters
 
 <div class="params">
-<div class="parameter"><span class="param">projectId: number</span><p>The id of the project to list assets from</p></div>
-<div class="parameter"><span class="param">branchId: string</span><p>The id of the branch</p></div>
+<div class="parameter"><span class="param">projectId: number</span><p>The id of the project to list assets from.</p></div>
+<div class="parameter"><span class="param">branchId: string</span><p>The id of the branch.</p></div>
+<div class="parameter"><span class="param">skip [optional]: number</span><p>Number of assets to skip before listing. Used for pagination. Defaults to 0.</p></div>
+<div class="parameter"><span class="param">limit [optional]: number</span><p>Maximum number of assets to list. Defaults to 16. Maximum 100000.</p></div>
```

Comment:
Number 100000 is taken from Zach's code in playcanvas-sync https://github.com/playcanvas/playcanvas-sync/blob/12b8c40cd08bea9968ae2f8653619528505c4ab4/src/load-assets.js#L28